src/main/java/org/jabref/gui/JabRefDialogService.java

```diff
@@ -126,6 +127,7 @@ public static String shortenDialogMessage(String dialogMessage) {
         choiceDialog.setHeaderText(title);
         choiceDialog.setTitle(title);
         choiceDialog.setContentText(content);
+        Globals.getThemeLoader().installCss(choiceDialog.getDialogPane().getScene(), Globals.prefs);
```

Comment:
We are trying to remove the `Globals` class (in the long term). To facilitate this, could you please extract the `themeLoader` and `preferences`, and initialize them by passing the global objects as dependencies through the constructor (i.e. move `Globals` from this class to the caller)

Reply:
I have refactored the code but I m not sure if I've done it correctly